src/main/proto/cosmas.proto

```diff
@@ -18,6 +18,28 @@ syntax = "proto3";
 option java_package = "com.bardsoftware.papeeria.backend.cosmas";
 option java_outer_classname = "CosmasProto";
 
+message FileVersionInfo {
+    int64 generation = 1;
+    int64 timestamp = 2;
+}
+
+message FileCoffin {

```

Comment:
I like the word, but "thomb" or "thombstone" is used way more often in software systems.

Reply:
done